forge-gui/res/cardsfolder/upcoming/ballroom_brawlers.txt

```diff
@@ -0,0 +1,14 @@
+Name:Ballroom Brawlers
+ManaCost:3 W W
+Types:Creature Human Warrior
+PT:3/5
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigSelectTargetCreature | TriggerDescription$ Whenever CARDNAME attacks, CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.
+SVar:TrigSelectTargetCreature:DB$ Pump | ValidTgts$ Creature.YouCtrl+Other | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one other target creature | AITgts$ Creature.withoutFirst Strike | SubAbility$ DBKeywordChoice | SpellDescription$ CARDNAME and up to one other target creature you control both gain your choice of first strike or lifelink until end of turn.
+SVar:DBKeywordChoice:DB$ GenericChoice | Defined$ You | Choices$ DBFirstStrike,DBLifelink | AILogic$ DBFirstStrike
+SVar:DBFirstStrike:DB$ Pump | Defined$ Self | KW$ First Strike | SubAbility$ DBFirstStrike2 | SpellDescription$ First strike

```

Comment:
Could you try this and the Lifelink as a PumpAll with `ValidCards$ Card.Self,Targeted` ??

I don't think we can do `Defined$ This,That` in a regular Pump yet

Reply:
I tried both PumpAll Defined$ This,That and Defined$ This & That, neither worked :C

Reply:
Alright, I corrected to pumpall ValidCards$ Card.Self,Targeted, however I get a NPE (Error in Trigger for Card)
I think the target messes with pumpall.
This looks like a headache to fix
[NPE.txt](https://github.com/Card-Forge/forge/files/8484150/NPE.txt)

Reply:
@tool4ever any thoughts regarding this?

Reply:
Can you try `ValidCards$ Card.Self,Targeted.Self` ?

Reply:
> Can you try `ValidCards$ Card.Self,Targeted.Self` ?

:kekw: the NPE was because of a typo in the "SVar"
However, I fixed it and while it doesn't crash the game - it still doesn't work, it only adds the keyword to itself. I also tried different variations, with and without Self, even with remember pump.

Reply:
Ugh, thanks for testing all that. I thought for sure it would work.